LaGeSo Audit Results

sven-pa opened the pull request:
Again, as discussed, watch out for the following changes in document names:
* Renamed stakeholder requirements to user needs (this affects the list and review checklist❗️)
* Renamed SOP Incident Reporting -> SOP Vigilance

We also discussed to add an "audited" label to some of the templates on our website, close to the download button (@olieidel). After implementing the changes, this label could be applied to the following templates:

[SOP Update of Regulations](https://github.com/openregulatory/templates/blob/master/templates/qms/sop-update-of-regulations.md)
[SOP Vigilance](https://github.com/openregulatory/templates/blob/master/templates/qms/vigilance/sop-incident-reporting.md)
[Clinical Evaluation Report](https://github.com/openregulatory/templates/blob/master/templates/clinical_evaluation/clinical-evaluation-report.md)
[Risk Management Plan](https://github.com/openregulatory/templates/blob/master/templates/techdoc/14971/risk-management-plan.md)
[Risk Table](https://github.com/openregulatory/templates/blob/master/templates/techdoc/14971/risk-table-fmea.md)
[SOP Post-Market Surveillance](https://github.com/openregulatory/templates/blob/master/templates/qms/post-market-surveillance/sop-post-market-surveillance.md)
[Post-Market Surveillance Plan](https://github.com/openregulatory/templates/blob/master/templates/qms/post-market-surveillance/post-market-surveillance-plan.md)
[Periodic Safety Update Report](https://github.com/openregulatory/templates/blob/master/templates/qms/post-market-surveillance/periodic-safety-update-report.md)

sven-pa committed:
Define input channels

Added input channels (para. 1)

sven-pa committed:
Added role and responsibilities for PRRC (Art. 15 MDR)

sven-pa committed:
Evaluate implementation of regulatory updates

sven-pa committed:
QMM-QM: PRRC Qualifications

sven-pa committed:
QMM-QM: fixed typo

sven-pa committed:
URG-SOP: fixed typos

ivoflipse commented on templates/qms/quality-manual-policy-objectives.md:
```diff
@@ -102,6 +102,23 @@ The Quality Management Officer (QMO) is responsible to:
+The PRRC shall not be subjected to Management instructions while carrying out his/her responsibilities specified above. His/her tasks may be delegated to other roles as long as it is ensured that final responsibility stays with the PRRC. She or he has the power and authority to represent the company in the scope of his/her responsibilities, e.g. in communicating with state authorities.
+
+Required qualification for this role:
+* Fluent in German and English language
```
Only in Germany right?

sven-pa replied:
Well spotted, thanks. Will update.

sven-pa committed:
Updated PRRC language requirements

sven-pa committed:
Added references to CEV / VIG

LageSo Audit rework. Para. 1: formulation fixes, reference to CEV-TPL-PMCFP and reference to VIG-SOP

sven-pa committed:
Updated file name, rework from audit findings

UPDATED FILE NAME | Updated contact information of competent authorities (avoiding hyperlinks), immediate action as part of first step, FSCA must be reported to authorities

sven-pa committed:
VIG-SOP: Numbering of sub-paragraphs

Corrected numbering of sub-paragraphs

sven-pa committed:
VIG-INA: updated serious incident terminology and MDSO/PRRC

sven-pa committed:
MDD/MDR and referenced UDI-PI to SWD-SOP

Updated MDD/MDR | Updated for UDI-PI -> use software version, referenced SWD-SOP

sven-pa committed:
CHM-SOP: Audit rework

Lageso inspection: Harmonized terminology for changes (substantial -> significant)

sven-pa committed:
SWD-SOP Audit Rework

Lageso audit: changed risk mitigation (ALARP -> AFAP), prioritization of risk mitigation measures (para. 3) | Added typical versioning with reference to SWDP (para. 4) |  Terminology: usage scenarios -> user needs (following FDA) | Added list and review checklist for user needs review, corrected doc IDs |

sven-pa committed:
STR -> User needs

(following FDA)

sven-pa committed:
STR -> UN

(following FDA)

sven-pa committed:
STR -> User needs (groups)

sven-pa committed:
CEV Report: Lageso Audit Rework

Adjust equivalence score without biological category

sven-pa committed:
RMP: Lageso audit

Updated ALARP -> AFAP and prioritization of risk control measures (para. 3)

sven-pa committed:
DRC-SOP: remark on QMS doc location

sven-pa committed:
Risk Table: New Risks by Mitigation

Audit rework: add column for documenting new risks introduced by mitigation measures

sven-pa committed:
Comment on quality objectives

Revised instruction for quality objectives

sven-pa committed:
Instructions to define risk acceptance matrix and perform risk analysis

Lageso audit: described how to define risk acceptance matrix and how to perform risk analysis (para. 2+3) in more detail.

sven-pa committed:
Reference to risk acceptance matrix

Replaced by reference to risk management plan instead

sven-pa committed:
Delete risk acceptance matrix

Lageso: Moved this into risk management plan

sven-pa committed:
RMP: added risk acceptance matrix

Lageso audit: moved the risk acceptance matrix from the risk table to the risk management plan

sven-pa committed:
PMS-SOP: process references

Added SOP references based on Lageso audit

sven-pa committed:
Acceptance criteria for complaint rate

Lageso: SOP references + acceptance criteria for overall complaint rate

sven-pa committed:
Acceptance criteria for complaint rate #2

sven-pa committed:
Criteria for complaint rate

sven-pa committed:
Keywords for Similar Devices

sven-pa committed:
Usage scenarios -> user needs

olieidel committed:
minor formatting improvements

olieidel reviewed (approved):
Thanks so much for your work! 🙂
